cloudman/data/update_keycloak.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
+
+# abort if any command fails
+set -e
+username="admin"
+password="{{ .Values.keycloak.keycloak.password }}"
+
+# get auth token
+token=$(curl -k -s -d "client_id=admin-cli" -d "username=admin" -d "password=$password" -d "grant_type=password" \
+       "https://{{ .Values.global.domain }}/auth/realms/master/protocol/openid-connect/token" | jq -r '.access_token')
+
+# get admin user id
+user_id=$(curl -k -s -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master/users/?username=admin | \
+          jq -r '.[] | select(.username=="admin") | .id')
+
+# update admin user info
+updated_user=$(curl -k -s -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master/users/$user_id | \
+          jq -r '.firstName="Alex" | .lastName="ITNEW" | .email="newtest@me.org"')
```

Comment:
Looks like some test data slipped through. Should be `'.firstName="{{ .Values.admin_firstname }}`

Reply:
Since you're in this headspace of keycloak ReST requests, I think it would make sense to continue with our original plan, and move this script to an init container instead of having it as a post-install hook. Otherwise, revisiting this would require recovering a lot of context, whereas now it seems like a few more simple changes. In this line, we just need an additional  check here where the details are only updated if the email is empty.

Reply:
Done in https://github.com/almahmoud/cloudman-helm/commit/f76baa9816521e35c6f49003998885377e5c97de and https://github.com/CloudVE/cloudlaunch-helm/pull/4